src/config/index.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
     allow_ad_personalization_signals: false,
     debug_mode: IS_DEVELOPMENT,
   },
+  STABILITY_API_KEY: process.env.STABILITY_API_KEY || "test",
```

Comment:
does "test" api key work? if not we can install dotenv and use tradetrust api key from there.

Reply:
hey it doesn't, let me remove it